src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -21,7 +21,7 @@ public static IEnumerable<object[]> InputData()
     }
 
     [Theory]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Issue https://github.com/dotnet/corefx/issues/18220")]
+    [ActiveIssue("https://github.com/dotnet/corefx/issues/18220", TargetFrameworkMonikers.UapAot)]
```

Comment:
Personally, I like the url form. We have three different repos, plus TFS, plus the convenience of being able to go straight to the issue by clicking. And it's still reasonably parsable by tooling if desired.